waku/handshake.go

```diff
@@ -18,14 +18,61 @@ import (
 // In the case of RLP, options should be serialized to an array of tuples
 // where the first item is a field name and the second is a RLP-serialized value.
 type statusOptions struct {
-	PoWRequirement       uint64      `rlp:"key=0"` // RLP does not support float64 natively
+	PoWRequirement       *uint64     `rlp:"key=0"` // RLP does not support float64 natively
 	BloomFilter          []byte      `rlp:"key=1"`
-	LightNodeEnabled     bool        `rlp:"key=2"`
-	ConfirmationsEnabled bool        `rlp:"key=3"`
-	RateLimits           RateLimits  `rlp:"key=4"`
+	LightNodeEnabled     *bool       `rlp:"key=2"`
+	ConfirmationsEnabled *bool       `rlp:"key=3"`
+	RateLimits           *RateLimits `rlp:"key=4"`
 	TopicInterest        []TopicType `rlp:"key=5"`
 }
 
+func (s statusOptions) WithoutDefaults() statusOptions {
+	if s.PoWRequirement != nil && *s.PoWRequirement == 0 {
+		s.PoWRequirement = nil
+	}
+
+	if s.LightNodeEnabled != nil && !*s.LightNodeEnabled {
+		s.LightNodeEnabled = nil
+	}
+
+	if s.ConfirmationsEnabled != nil && !*s.ConfirmationsEnabled {
+		s.ConfirmationsEnabled = nil
+	}
+
+	if s.RateLimits != nil && s.RateLimits.IsZero() {
+		s.RateLimits = nil
+	}
+
+	return s
+}
+
+func (s statusOptions) WithDefaults() statusOptions {
+	if s.PoWRequirement == nil {
+		var pow uint64 = 0
```

Comment:
How about we set the default of the peer its PoW to the same value as the node its default? 
Or some other default value, but not 0?

Reply:
I would probably set it to a default rather than the host PoW, just to make it a bit simpler, we could use the current minpow used by the nodes? (can't remember exactly what it is, 0.0002 or something?)

Reply:
What we use is practically 0. Especially in terms of calculating it. We run some tests once and I think anything below 0.2 or 0.02 was close to 0. We use [`0.002`](https://github.com/status-im/status-go/blob/44aa31398107f2dcb224cf6e689896aed0079227/protocol/transport/message.go#L9).

Reply:
> I would probably set it to a default rather than the host PoW, just to make it a bit simpler, we could use the current minpow used by the nodes? (can't remember exactly what it is, 0.0002 or something?)

That's what I meant.